src/server/shell/basic_accessibility_manager.cpp

```diff
@@ -32,7 +33,7 @@ inline Transformer* mir::shell::BasicAccessibilityManager::Registration<Transfor
 }
 
 template <typename Transformer>
-inline void mir::shell::BasicAccessibilityManager::Registration<Transformer>::remove_registration() const
+inline bool mir::shell::BasicAccessibilityManager::Registration<Transformer>::remove_registration() const
```

Comment:
Why change the signature? The return value isn't useful, nor used

Reply:
Correction: it is used in `BasicAccessibilityManager::simulated_secondary_click_enabled`